appinventor/docs/reference/blocks/math.html

```diff
@@ -386,6 +395,60 @@ <h3 id="isnumber">is a number</h3>
 <h3 id="convertnumber">convert number</h3>
 <p><img src="images/math/convertnumber.png" /></p>
 <p>Takes a text string that represents a positive integer in one base and returns a string that represents the same number is another base.  For example, if the input string is 10, then converting from base 10 to binary will produce the string 1010;  while if the input string is the same 10,  then converting from binary to base 10 will produce the string 2.   If the input string is the same 10, then converting from base 10 to hex will produce the string A.</p>
+<h3 id="bitwise_and">Bitwise And</h3>
+<p><img src="images/math/bitwise_and.png" /></p>
+<p>Takes two numbers and compares each pair of bits. Each bit of the result is true only if the corresponding bits of both operands are true.</p>
```

Comment:
Minor change here: rather than stating that bits are true or false, let's say that they are 0 or 1 since that is what the table shows. Also, this way it does not conflicting with true/false in logical and/or.